ods_ci/tests/Tests/0100__platform/0101__deploy/0104__operators/0104__rhods_operator/0113__dsc_components.robot

```diff
@@ -6,321 +6,548 @@ Resource            ../../../../../Resources/OCP.resource
 
 Validate ModelRegistry Removed State
     [Documentation]    Validate that ModelRegistry management state Removed does remove relevant resources.
-    [Tags]    Operator    Tier1    RHOAIENG-10404    modelregistry-removed      Integration
-
+    [Tags]
+    ...    Operator
+    ...    Tier1
+    ...    RHOAIENG-10404
+    ...    modelregistry-removed
+    ...    Integration
     # Properly validate Removed state by first setting to Manged, which will ensure that namspace
     # was created as needed for later validating that namespace persisted when component is Removed
-    [Setup]   Set DSC Component Managed State And Wait For Completion   modelregistry    ${MODELREGISTRY_CONTROLLER__DEPLOYMENT_NAME}    ${MODELREGISTRY_CONTROLLER__LABEL_SELECTOR}
+    [Setup]     Set DSC Component Managed State And Wait For Completion     modelregistry       ${MODELREGISTRY_CONTROLLER__DEPLOYMENT_NAME}        ${MODELREGISTRY_CONTROLLER__LABEL_SELECTOR}
 
-    Set DSC Component Removed State And Wait For Completion   modelregistry    ${MODELREGISTRY_CONTROLLER__DEPLOYMENT_NAME}    ${MODELREGISTRY_CONTROLLER__LABEL_SELECTOR}
+    Set DSC Component Removed State And Wait For Completion
+    ...    modelregistry
+    ...    ${MODELREGISTRY_CONTROLLER__DEPLOYMENT_NAME}
+    ...    ${MODELREGISTRY_CONTROLLER__LABEL_SELECTOR}
 
     # Note: Model Registry namespace will not be deleted when component state changed from Manged to Removed
     Check Model Registry Namespace
 
-    [Teardown]     Restore DSC Component State    modelregistry    ${MODELREGISTRY_CONTROLLER__DEPLOYMENT_NAME}    ${MODELREGISTRY_CONTROLLER__LABEL_SELECTOR}    ${SAVED_MANAGEMENT_STATES.MODELREGISTRY}
+    [Teardown]      Restore DSC Component State     modelregistry       ${MODELREGISTRY_CONTROLLER__DEPLOYMENT_NAME}        ${MODELREGISTRY_CONTROLLER__LABEL_SELECTOR}     ${SAVED_MANAGEMENT_STATES.MODELREGISTRY}
 
 Validate KServe Controller Manager Managed State
     [Documentation]    Validate that the DSC KServe Controller Manager component Managed state creates the expected resources,
     ...    check that KServe Controller Manager deployment is created and pod is in Ready state
-    [Tags]    Operator    Tier1    RHOAIENG-7217    kserve-controller-manager-managed      Integration
-
-    Set DSC Component Managed State And Wait For Completion   kserve    ${KSERVE_CONTROLLER_MANAGER_DEPLOYMENT_NAME}    ${KSERVE_CONTROLLER_MANAGER_LABEL_SELECTOR}
-
-    [Teardown]     Restore DSC Component State    kserve    ${KSERVE_CONTROLLER_MANAGER_DEPLOYMENT_NAME}    ${KSERVE_CONTROLLER_MANAGER_LABEL_SELECTOR}    ${SAVED_MANAGEMENT_STATES.KSERVE}
+    [Tags]
+    ...    Operator
+    ...    Tier1
+    ...    RHOAIENG-7217
+    ...    kserve-controller-manager-managed
+    ...    Integration
+    ...    ExcludeOnODH
+    ...    ExcludeOnODHCommunity
+    ...    ExcludeOnRHOAI
```

Comment:
Why is this exclude on RHOAI? This way we won't be able to test KServe Managed state.

Reply:
Talked with @asanzgom and this will be addressed in the followup PR.